platform/security/policy/security-policy-context/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -19,6 +19,11 @@
          name="Web Context Policy Manager"
          id="org.codice.ddf.security.policy.context.impl.PolicyManager">
 
+        <AD description="Depth to which paths will be traversed. Any value greater than 500 will be set to 500."
```

Comment:
Yeah, I was just trying to come up with a value that was large enough to cover any web app that someone might want to deploy and also small enough that it wouldn't cause any issues. Plus we're consuming the slashes now instead of recursing on them, so those will just be tossed. So that is why I picked 500. I also figured 20 was plenty for a default value of anything we're likely to encounter.